### PR TITLE
Add a default key to the permdef_schema

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: synapse/lib/storm.py [hunk 1]
```diff
@@ -751,8 +752,12 @@
                 if $pdef.perms {
                     $lib.print("Package ({name}) defines the following permissions:", name=$cmdopts.name)
                     for $permdef in $pdef.perms {
-                        $permtext = $lib.str.join('.', $permdef.perm).ljust(32)
-                        $lib.print("{permtext} : {desc}", permtext=$permtext, desc=$permdef.desc)
+                        $text = `{$lib.str.join('.', $permdef.perm).ljust(32)} : {$permdef.desc}`
+                        $default = $permdef.default
+                        if $default {
+                            $text = `{$text} ( default: True )`
```
Comment (invisig0th) [inline synapse/lib/storm.py:758]: Should probably start using JSON `true` or `$lib.true` rather than python syntax in output

Comment (invisig0th) [reply synapse/lib/storm.py:758]: Also, we should probably just cast `$permdef.default` as a bool and print it either way

Comment (vEpiphyte) [reply synapse/lib/storm.py]: don't want to cast to a bool and print that, since the system mode value is an integer.

Comment (vEpiphyte) [reply synapse/lib/storm.py]: just always printing and relying on the updated torepr() behavior now

**Pull request merged**